collect_app/src/main/java/org/odk/collect/android/activities/GoogleDriveActivity.java

```diff
@@ -789,6 +789,7 @@ void setTaskListener(TaskListener tl) {
                         .setFields("id")
                         .execute().getId();
             } catch (UserRecoverableAuthIOException e) {
+                createAlertDialog(getString(R.string.auth_failed_drive));
```

Comment:
@dcbriccetti Is that really the exception you had in mind? I can trigger that one by revoking access but I see an oauth prompt that I can use to recover from the problem.